cmd/skopeo/utils.go

```diff
@@ -180,12 +184,23 @@ func (opts *imageOptions) newSystemContext() (*types.SystemContext, error) {
 	if opts.credsOption.present && opts.noCreds {
```

Comment:
Shouldn’t this now check for `opts.credsOption.present || opts.userName.present) && opts.noCreds` (with an appropriate error message update)?

Or in some other way ensure that of the three options, exactly one can be set.

Reply:
Good catch. I had missed that. I've now updated this PR to fix that.